qa/evil-tests/build.gradle

```diff
@@ -40,36 +40,12 @@ apply plugin: 'opensearch.testclusters'
 apply plugin: 'opensearch.standalone-test'
 
 dependencies {
-  testImplementation 'com.google.jimfs:jimfs:1.3.1'
```

Comment:
Do you think its worth looking into https://github.com/autonomousapps/dependency-analysis-gradle-plugin?